uk_bin_collection/uk_bin_collection/councils/WestDevonBoroughCouncil.py

```
@@ -0,0 +1,172 @@
+                        matched_uprn = entry[0]
+                        break
+
+                if not matched_uprn:
+                    for entry in addresses.values():
+                        addr_text = entry[1].lower().strip()
+                        if paon_lower in addr_text:
+                            matched_uprn = entry[0]
+                            break
+
+                if not matched_uprn:
+                    first_entry = list(addresses.values())[0]
```

Ruff (0.15.12)

[warning] Prefer `next(iter(addresses.values()))` over single element slice

Replace with `next(iter(addresses.values()))`

(RUF015)